ui-showdetails/src/main/java/app/tivi/showdetails/details/ShowDetails.kt

```diff
@@ -133,12 +130,14 @@ import app.tivi.data.resultentities.numberToAir
 import app.tivi.data.resultentities.numberWatched
 import app.tivi.data.views.FollowedShowsWatchStats
 import dev.chrisbanes.accompanist.coil.CoilImage
-import dev.chrisbanes.accompanist.insets.AmbientWindowInsets
+import dev.chrisbanes.accompanist.insets.LocalWindowInsets
 import dev.chrisbanes.accompanist.insets.navigationBarsPadding
 import dev.chrisbanes.accompanist.insets.statusBarsHeight
 import org.threeten.bp.OffsetDateTime
 
-val AmbientShowDetailsTextCreator = staticAmbientOf<ShowDetailsTextCreator>()
+val LocalShowDetailsTextCreator = staticCompositionLocalOf<ShowDetailsTextCreator> {
```

Comment:
This doesn't seem to pass the rough test for what should be a `CompositionLocal`:
1. Has a sensible default value
2. Is widely applicable but tree scoped.

Can this just be passed down as a parameter?

Reply:
Yep, I'm going to fix all of these once I move everything to Compose. It's all a bit of a mess right now because it's injected.

The rough plan I have is to combine all of the `TextCreator`s into one global instance.